### DISCUSSION TITLE
CLIP Skip

### DISCUSSION THREAD

**Discussion by kalle07** [mcmonkeyprojects/SwarmUI#377] [Q&A] [answered]
where i set 
CLIP Skip ?
maybe thats one reason for different images same seed ?
maybe Positive/Neg ADM Guidance Scaler different ? what else ?

**Comment by kwal559** [comment]
bottom left 'advanced sampling'.  You'll also notice a hidden gem in that list, a background remover.  That said, it would be nice to have a toggle to reset after use.  I leave it on by accident all the time.

**Comment by mcmonkey4eva** [comment, marked as answer]
you can also type "clip" into the param filter at top left and you'll be narrowed down to just a few relevant params, including clip stop at last layer (aka clip skip)
